src/bridge/cockpitdbusjson.c

```diff
@@ -2674,17 +2674,7 @@ on_name_appeared (GDBusConnection *connection,
 {
   CockpitDBusJson *self = COCKPIT_DBUS_JSON (user_data);
 
-  g_object_ref (self);
-
-  if (!self->default_appeared)
```

Comment:
Please also remove the variable from the struct, since it is now unused.

Reply:
Good catch, thanks.